_posts/2020-06-21-one-stop-guide-to-lombok.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "One-stop Guide to Lombok"
+date: 2020-06-21 10:00 +0500
+modified: 2020-06-21 10:15 +0500
+---
+
+# What is it
+Lombok is a Java library which makes Java less chatty whilst making your code more readable. Java is great language but when you use its features you have to write a lot of boilerplate code.
+
+# Why to use Lombok
+
+Lombok helps you with a set of annotations to make your code more readable without losing its functionality. You don’t need to write typical constructors, setters or getters. With the help of Lombok you can easily add logging feature to you class or create fully functional builder.
```

Comment:
This section doesn't really say why I should use Lombok. It only says what the following sections are saying anyways.

Imagine the reader doesn't know Lombok and is skeptical. What would you tell them? For example something like "you can reduce the codebase by 30%" (with a link to an article that proves this). Link to other articles that explain the why. Please do some more research here.